.utility/update-docs-repo.sh

```diff
@@ -11,7 +11,7 @@ git subtree --prefix=docs/ split -b documentation
 
 cd $HOME
 
-git clone --quiet --branch=master git@github.com:loklak/dev.loklak.org.git loklak_docs
```

Comment:
It authenticates the puller to push to the repository because he is now a special user that Github recognises (as they have the key).